yt_dlp/compat/__init__.py

```diff
@@ -83,12 +85,46 @@ def compat_expanduser(path):
 WINDOWS_VT_MODE = False if compat_os_name == 'nt' else None
 
 
-def windows_enable_vt_mode():  # TODO: Do this the proper way https://bugs.python.org/issue30075
+def set_windows_conout_mode(new_mode, mask=0xffffffff):
+    # based on https://github.com/python/cpython/issues/74261#issuecomment-1093745755
```

Comment:
Resolved? This is basically copied verbatim.

Reply:
And I changed PR message, is this not public domain?

Reply:
Not unless @eryksun is willing to release it into PD

Reply:
I couldn't contact him, https://bugs.python.org/user16413.

Reply:
> mannequin

So it’s not even necessarily the same as the GitHub user of the same name.

Reply:
> > mannequin
> 
> So it’s not even necessarily the same as the GitHub user of the same name.

It is. See https://bugs.python.org/user16413. (Otherwise the account wouldn't have linked)

Reply:
I just followed original https://bugs.python.org/issue30075 link under `Show more details`.

Reply:
PSF-2.0 is based on BSD.